ci/run_black.ps1

```diff
@@ -1,16 +1,17 @@
 $ExitCode = 0
-$GitDiff = git diff --name-only origin/master | Select-String -Pattern ".*\.py" | Select-String -NotMatch ".*/project_template/.*"
+$GitDiff = git diff --name-only origin/master
+$GitDiffBlack = git diff --name-only origin/master | Select-String -Pattern ".*\.py" | Select-String -NotMatch ".*/project_template/.*"
```

Comment:
Don't-Repeat-Yourself. You could do `Write-Output $GitDiff` instead of running `git diff` twice.